src/sqs/backfill.test.ts

```diff
@@ -37,15 +71,50 @@ describe("backfill", () => {
 		lastAttempt: false
 	};
 
-	afterEach(() => {
-		sentryCaptureExceptionMock.mockReset();
+	describe("Sentry", () => {
```

Comment:
This is the existing test, I just wrapped it in a describe for house keeping and added some mocks in a beforeeach